lntest/itest/lnd_test.go

```diff
@@ -4227,21 +4238,29 @@ func channelForceClosureTest(net *lntest.NetworkHarness, t *harnessTest,
 		t.Fatalf(predErr.Error())
 	}
 
-	// At this point, Bob should now be aware of his new immediately
+	// At this point, Carol should now be aware of her new immediately
 	// spendable on-chain balance, as it was Alice who broadcast the
 	// commitment transaction.
 	ctxt, _ = context.WithTimeout(ctxb, defaultTimeout)
-	carolBalResp, err = net.Bob.WalletBalance(ctxt, carolBalReq)
-	if err != nil {
-		t.Fatalf("unable to get carol's balance: %v", err)
-	}
-	carolExpectedBalance := btcutil.Amount(carolStartingBalance) + pushAmt
-	if btcutil.Amount(carolBalResp.ConfirmedBalance) < carolExpectedBalance {
-		t.Fatalf("carol's balance is incorrect: expected %v got %v",
-			carolExpectedBalance,
-			carolBalResp.ConfirmedBalance)
+	carolBalResp, err = carol.WalletBalance(ctxt, carolBalReq)
+	require.NoError(t.t, err, "unable to get carol's balance")
+
+	// Carol's expected balance should be its starting balance plus the
+	// push amount sent by Alice and minus the miner fee paid.
+	carolExpectedBalance := btcutil.Amount(carolStartingBalance) +
+		pushAmt - totalFeeCarol
+
+	// In addition, if this is an anchor-enabled channel, further add the
+	// anchor size.
+	if channelType == commitTypeAnchors {
+		carolExpectedBalance += btcutil.Amount(anchorSize)
 	}
 
+	require.Equal(t.t, carolExpectedBalance,
```

Comment:
micro nit: insert line break before `t.t`.

Reply:
Done!

Reply:
No I meant after the parenthesis, before the first argument, since it's a multi line function call where we normally put all arguments on  a new line. But not too important, just my code OCD getting triggered a bit :see_no_evil:

Reply:
Hah yeah ofc, somehow I was thinking about the log lines. Updated since it's our consensus that this is the preferred style🤓